src/actions/trending.js

```diff
@@ -0,0 +1,18 @@
+import { createAction } from 'redux-actions';
+
+const TRENDING_GIFS_GET = 'TRENDING_GIFS_GET';
+const TRENDING_GIFS_FETCHED = 'TRENDING_GIFS_FETCHED';
+
+const getTrendingGifs = (payload = {}) => ({ type: TRENDING_GIFS_GET, payload });
+const fetchedTrendingGifs = createAction(TRENDING_GIFS_FETCHED);
```

Comment:
Any reason why you are mixing the creators declaration between `createAction` and manual?